core/collection-factory/mapdb3/src/main/java/org/eclipse/rdf4j/collection/factory/mapdb/MapDb3CollectionFactory.java

```diff
@@ -69,9 +74,10 @@ private MapDb3BackedQueue(Map<Long, T> m) {
 
 		@Override
 		public boolean offer(T arg0) {
-			m.put((Long) tail++, arg0);
-			if (tail % iterationCacheSyncThreshold == 0)
+			m.put(tail++, arg0);
+			if (tail % iterationCacheSyncThreshold == 0) {
```

Comment:
Sorry for the `{}` less if branches, that is the style at my work. I dislike it but got used to it.